test/bitarray.jl

```diff
@@ -1114,7 +1114,7 @@ timesofar("nnz&find")
     b1 = trues(v1)
     b2 = falses(v1)
     for i = 1:v1
-        @test findprev(b1, i) == findprev(b1, true, i) == findprev(identity, b1, i)
```

Comment:
Why not test `findprev(equalto(true), b1, i)`?

Reply:
The same codepath is covered by `identity`.

Reply:
I don't think so, given `findnext(pred::EqualTo, B::BitArray, start::Integer)` now exists.